manager/subscription/tests/test_heartbeat.py

```diff
@@ -238,3 +258,96 @@ async def test_unauthorized_commander(self):
+
+        hb_manager = HeartbeatManager()
+        # Arrange
+        await hb_manager.reset()
+        communicator = WebsocketCommunicator(application, self.url)
+        connected, subprotocol = await communicator.connect()
+
+        assert connected == True
+
+        # Act 1 (Subscribe)
+        msg = {
+            "option": "subscribe",
+            "category": "heartbeat",
+            "csc": "manager",
+            "salindex": 0,
+            "stream": "stream",
+        }
+        await communicator.send_json_to(msg)
+        response = await communicator.receive_json_from()
+
+        # Assert 1
+        assert (
+            response["data"] == f"Successfully subscribed to heartbeat-manager-0-stream"
+        )
+
+        response = await communicator.receive_json_from(timeout=10)
+        assert response["data"][0]["data"]["timestamp"] is not None
+
+        # Act 2 Set producer heartbeat
+        timestamp = datetime.datetime.now().timestamp()
+        hb_manager.set_heartbeat_timestamp("Commander", timestamp)
+        response = await communicator.receive_json_from(timeout=4)
+
+        # Assert 2
+        heartbeat_sources = [source["csc"] for source in response["data"]]
+        assert "Commander" in heartbeat_sources
+        await communicator.disconnect()
+        await hb_manager.stop()
+
+    @pytest.mark.asyncio
+    @pytest.mark.django_db(transaction=True)
+    @override_settings(HEARTBEAT_QUERY_COMMANDED=False)
+    async def test_heartbeat_commander_without_heartbeat_query_commanded(self):
+
+        hb_manager = HeartbeatManager()
+        # Arrange
+        await hb_manager.reset()
+        communicator = WebsocketCommunicator(application, self.url)
+        connected, subprotocol = await communicator.connect()
+
+        assert connected == True
+
+        # Act 1 (Subscribe)
+        msg = {
+            "option": "subscribe",
+            "category": "heartbeat",
+            "csc": "manager",
+            "salindex": 0,
+            "stream": "stream",
+        }
+        await communicator.send_json_to(msg)
+        response = await communicator.receive_json_from()
+
+        # Assert 1
+        assert (
+            response["data"] == f"Successfully subscribed to heartbeat-manager-0-stream"
+        )
+
+        try:
+            response = await communicator.receive_json_from(timeout=10)
+            #assert response["data"][0]["data"]["timestamp"] is not None
+
+            # Act 2 Set producer heartbeat
+            timestamp = datetime.datetime.now().timestamp()
+            hb_manager.set_heartbeat_timestamp("Commander_FAKE", timestamp)
+            response = await communicator.receive_json_from(timeout=4)
+
+            # Assert 2
+            heartbeat_sources = [source["csc"] for source in response["data"]]
+            assert "Commander_FAKE" not in heartbeat_sources
+
+        except asyncio.TimeoutError:
+            # Assert 2
+            assert True
```

Comment:
This should be just a `pass`.

Reply:
that assert is to confirm that the expected state is an asyncio.TimeoutError